src/pcapi/scripts/setup_app_search.py

```diff
@@ -0,0 +1,201 @@
+from pcapi.flask_app import app
+from pcapi.utils.human_ids import humanize
+
+
+HOST = "https://search-testing.ent.europe-west1.gcp.cloud.es.io"
+PRIVATE_API_KEY = os.environ["APP_SEARCH_PRIVATE_API_KEY"]
+
+ENGINE_NAME = "offers"
+ENGINE_LANGUAGE = None
+
+SCHEMA = {
+    "offer_author": "text",
+    "offer_category": "text",
+    "offer_date_created": "date",
+    "offer_dates": "date",
+    "offer_description": "text",
+    "offer_is_digital": "number",
+    "offer_is_duo": "number",
+    "offer_is_event": "number",
+    "offer_isbn": "text",
+    "offer_label": "text",
+    "offer_music_type": "text",
+    "offer_name": "text",
+    "offer_performer": "text",
+    "offer_prices": "number",
+    "offer_ranking_weight": "number",
+    "offer_show_type": "text",
+    "offer_speaker": "text",
+    "offer_stage_director": "text",
+    "offer_stocks_date_created": "text",
+    "offer_tags": "text",
+    "offer_times": "text",
+    "offer_thumb_url": "text",
+    "offer_type": "text",
+    "offerer_name": "text",
+    "venue_city": "text",
+    "venue_department_code": "text",
+    "venue_name": "text",
+    "venue_position": "geolocation",
+    "venue_public_name": "text",
+}
+
+
+# FIXME (dbaty): we're going to try without this default position.
+# DEFAULT_LONGITUDE = 2.409289
+# DEFAULT_LATITUDE = 47.158459
+
+
+def _call_api(method, endpoint, data):
+    headers = {"Authorization": f"Bearer {PRIVATE_API_KEY}", "Content-Type": "application/json"}
+    url = f"{HOST}{endpoint}"
+    data = json.dumps(data, cls=AppSearchJsonEncoder)
+    response = requests.request(method, url, headers=headers, data=data)
+    if not response.ok:
+        result = response.json()
+        try:
+            errors = [result["error"]]
+        except KeyError:
+            errors = result["errors"]
+        print("\n".join(f"ERR: {error}" for error in errors))
+        return False
+    return True
+
+
+def create_engine():
+    endpoint = "/api/as/v1/engines"
+    data = {"name": ENGINE_NAME, "language": ENGINE_LANGUAGE}
+    if _call_api("POST", endpoint, data):
+        print("Engine has been created.")
+
+
+def init_schema():
+    endpoint = f"/api/as/v1/engines/{ENGINE_NAME}/schema"
+    if _call_api("POST", endpoint, SCHEMA):
+        print("Schema has been initialized.")
+
+
+# FIXME : in tests, check that this function does not return any
+# boolean. Boolean values would be rejected by App Search API. They
+# must be converted as integers. It's not easy to do that at the
+# serialization level.
+def serialize(offer):
+    dates = []
+    if offer.isEvent:
+        dates = [stock.beginningDatetime.timestamp() for stock in offer.bookableStocks]
+    extra_data = offer.extraData
+    # FIXME: see Cyril's FIXME about that.
+    isbn = (extra_data.get("isbn") or extra_data.get("visa")) if extra_data else None
+
+    venue = offer.venue
+    if venue.longitude is not None and venue.latitude is not None:
+        position = [venue.longitude, venue.latitude]
```

Comment:
```suggestion
        position = f"{venue.latitude}, {venue.longitude}"
```

J'ai l'impression qu'App Search a du mal à detecter la geoloc:

![image](https://user-images.githubusercontent.com/10118284/123936253-4b343800-d995-11eb-806a-dfbde315f094.png)